src/d3a/models/strategy/pv.py

```diff
@@ -34,7 +34,7 @@ class PVStrategy(BaseStrategy, OfferUpdateFrequencyMixin):
                 help="Change the risk parameter. Valid values are between 1 and 100.")
     ]
 
-    parameters = ('panel_count', 'risk')
+    parameters = ('panel_count', 'risk', 'max_panel_power_W', 'final_selling_rate')
```

Comment:
Can you please add initial_selling_rate here as well (i will add it in the d3a-web as well)

Reply:
Will do, thanks!